### ISSUE TITLE
dr10574 - Migrate HO AVH data load to pipelines-jenkins

### ISSUE THREAD

**Issue by peggynewman** [AtlasOfLivingAustralia/data-management#684]
Ran twice monthly on 16th and 23rd

**Comment by nielsklazenga** [comment]
Depends on #689.

**Comment by javier-molina** [comment]
Already covered by https://github.com/AtlasOfLivingAustralia/preingestion/issues/180
